xamarin/Xamarin.Forms
# [iOS] Fix Background issue in Button CustomRenderer

### Description of Change ###

Fix Background issue in Button CustomRenderer in iOS.

### Issues Resolved ### 

- fixes #12372 

### API Changes ###
 
 None

### Platforms Affected ### 

- iOS

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 
![fix12372](https://user-images.githubusercontent.com/6755973/95321434-568d8380-089b-11eb-8fea-6aad4282bdd3.gif)

### Testing Procedure ###
Launch Core Gallery and navigate to the issue 12372. If you can see the background color of the custom Button below, the test has passed.

### PR Checklist ###
